javascript-light-bulb/main.js

```diff
@@ -1,5 +1,7 @@
+var lightOn = true;
+
 function lightSwitch(event) {
-  if (button.className === 'light on') {
+  if (button.classList.contains('on') === lightOn) {
```

Comment:
This is still reliant on the current class on the DOM. You should make this condition something like `if(lightOn === true)`. Keep in mind you will need to update the value of the variable to keep track of if the light is on right now or not every time the light changes.